src/games/strategy/triplea/oddsCalculator/ta/IOddsCalculator.java

```diff
@@ -52,5 +52,6 @@ AggregateResults setCalculateDataAndCalculate(final PlayerID attacker, final Pla
 
   void addOddsCalculatorListener(final OddsCalculatorListener listener);
 
+  // TODO: this method appears to never used.
```

Comment:
Out of scope for this PR, but marking it so we don't lose the information so easily.